span-normalizer/src/main/resources/configs/common/application.conf

```diff
@@ -4,41 +4,26 @@ service.admin.port = 8099
 main.class = org.hypertrace.core.spannormalizer.SpanNormalizer
 
 span.type = jaeger
-
-flink.job {
-  parallelism = 4
-  metrics {
-    metrics.reporters = "prometheus"
-    metrics.reporter.prometheus.class = "org.hypertrace.core.serviceframework.metrics.flink.PrometheusReporter"
-  }
-}
-
-flink.source {
-  type = kafka
-  topic = jaeger-spans
-  kafka {
-    bootstrap.servers = "localhost:9092"
-    bootstrap.servers = ${?KAFKA_BOOTSTRAP_SERVERS}
-    group.id = "jaeger-spans-to-raw-spans-job"
-    auto.offset.reset = "latest"
-    enable.auto.commit = true
-    auto.commit.interval.ms = 5000
-  }
+input.topic = "jaeger-spans"
+output.topic = "raw-spans-from-jaeger-spans"
+
+kafka.streams.config = {
+  application.id = jaeger-spans-to-raw-spans-job
```

Comment:
`span-normalizer`

Reply:
hmm i think this is used as a consumer group id so for continuity need to re-use that